src/lib/colorUtils.ts

```diff
@@ -1,4 +1,5 @@
-import * as Color from 'color'
+// @ts-ignore
+import Color from 'color'
```

Comment:
This module has the same issue as `classnames` :(

We need to make a decision about this.

Reply:
Done, this package was removed from dependencies